DOC-2496-openCypher syntax (#237)
tigergraph/gsql-docs: modules/openCypher-in-gsql/pages/openCypher-in-gsql.adoc

To see how to run openCypher on the TigerGraph platform, see xref:openCypher-in-gsql.adoc[].

[NOTE]
====

> From version *4.1.2*, certain openCypher syntaxes are now fully supported, eliminating previous restrictions.

Reviewer: ```suggestion
From version 4.1.2, certain restrictions on the `MATCH` and `WITH` clauses have been eliminated.
```
1. "syntax" already refers to a collection (multiple) rules, so plural "syntaxes" is not really correct here.
2. I think we still have one restriction on MATCH, so unfortunately we cannot say fully supported.
